pandapower/create.py

```diff
@@ -1844,6 +1859,80 @@ def create_gens(net, buses, p_mw, vm_pu=1., sn_mva=nan, name=None, index=None, m
 
     return index
 
+def create_motor(net, bus, pn_mech_mw, cos_phi=nan, vn_kv=nan, efficiency_percent=100.,
+                 loading_percent=100., name=None, lrc_pu=nan, scaling=1.0,
+                 rx=nan, index=None, in_service=True, cos_phi_n=nan,
+                 efficiency_n_percent=nan):
+    """
+    Adds a motor to the network.
+
+
+    INPUT:
+        **net** - The net within this motor should be created
+
+        **bus** (int) - The bus id to which the motor is connected
+        
+        **pn_mech_mw** (float) - Mechanical rated power of the motor
+
+
+    OPTIONAL:
+
+        **name** (string, None) - The name for this motor
+
+        **cos_phi** (float, nan) - cosine phi at current operating point
+
+        **cos_phi_n** (float) - cosine phi at rated power of the motor for short-circuit calculation
```

Comment:
I think I made it optional because it isn't needed for power flow, but not for short-circuit calculation. But this is not in line with the other create functions, where all power flow parameters are always required. I will make it required.